### DISCUSSION TITLE
Cleaning old data stored on OpenSearch

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#7522] [Q&A] [answered]
Dear all,

I would like to know if DIRAC automatically cleans old data stored on OpenSearch, both Monitoring and Job parameters data.
Or this should be handled directly on the OpenSearch side?
In both cases, do you have any advice on the policy for cleaning old data?

Thank you,

Luisa

**Comment by fstagni** [comment, marked as answer]
No, DIRAC does nor clean the old data stored in OpenSearch: for this, you need to specify policies directly in OpenSearch. We are not yet sharing them publicly, but at some point we could.

**Reply by arrabito** [reply]
OK thank you for your answer.
